massa-consensus-worker/src/controller.rs

```diff
@@ -281,6 +281,15 @@ impl ConsensusController for ConsensusControllerImpl {
                 .block_header_sender
                 .send(header.clone().content);
         }
+
+        if let Err(e) = self
+            .channels
+            .denunciation_factory_sender
+            .send(header.clone())
+        {
+            warn!("Cannot send header to denunciation factory: {}", e);
```

Comment:
Note that the consensus already has logic to detect double-staking: https://github.com/massalabs/massa/pull/3646/files#diff-08965c1c08d1437142b10be633e041151c0a7d8385e4d05363387c1a169e1204R115-R130

Reply:
discussed orally